web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/behandling/vedtak/dto/TotrinnskontrollAksjonspunkterDto.java

```diff
@@ -15,6 +15,7 @@ public class TotrinnskontrollAksjonspunkterDto {
     private Boolean totrinnskontrollGodkjent;
     private Set<VurderÅrsak> vurderPaNyttArsaker;
     private List<UttakPeriodeEndringDto> uttakPerioder;
+    //TODO TFP-5702 burde beslutter panelet vise om en behandling har endret dekningsgrad? De berørte behandlingene som er påvirket av avklaring på annen parts sak
```

Comment:
Er dette avledet av behandlingsResultat . konsekvens for ytelse?

Reply:
Har ikke laget egen konsekvens for endret dekningsgrad, men behandlingsresultatdto inneholder "endretDekningsgrad" felt som vi kan bruke